integration/tx_test.go

```diff
@@ -150,7 +150,7 @@ func TestGetTxViaGRPCGateway(t *testing.T) {
 		isTxBodyRetrieved = env.Exec("retrieve account addresses", steps, ExecRetry())
 	}()
 
-	env.Must(env.Serve("should serve", path, ExecCtx(ctx)))
+	env.Must(env.Serve("should serve", path, "", "", ExecCtx(ctx)))
```

Comment:
Not a priority, home dirs can passed as an Option to Serve.

Reply:
It was more simple to me to provide it as direct argument since `env.Serve` is scoped to tests